### PR TITLE
OPSEXP-1615 update readme.md with available java versions

### PR DESCRIPTION
**Pull request by piraartur**
Updated java legacy versions available to build from Dockerfile in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,17 +92,17 @@ The `alfresco-docker-base-java` image can be generated in multiple flavors by mi
 
 #### Java
 
-Either Java 8 (supported up to 5.2 and 6.0) or Java 11 can used used to build images using the `JAVA_MAJOR` build argument.
+Either Java 11 or Java 17 can used used to build images using the `JAVA_MAJOR` build argument.
```
Comment (mteodori) [inline README.md:95]: we can fix the existing error "can used used" to "can be used"

Comment (piraartur) [reply README.md]: > we can fix the existing error "can used used" to "can be used"

Fixed it in new commit

**Comment by piraartur** [comment]
> Can you also update the markdown table?

Yes, but If I could access shields.io and get to know proper links, I would update them also

**Comment by gionn** [comment]
> shields.io

just construct the new img url using the proper docker image tag you see on https://hub.docker.com/r/alfresco/alfresco-base-java/tags

e.g. https://img.shields.io/docker/image-size/alfresco/alfresco-base-java/jre17-rockylinux8

**Review by alxgomz — CHANGES_REQUESTED** [review]
Sorry I think I've been unclear.
java. 17 comes in addition to java 11 not as a replacement (we will continue publishing jre11 images as long as we support versions that use it).
Can you please reflect that in your changes?

**Comment by alxgomz** [comment]
take a look at the github actions workflow to better understand what's built from this repo

**Comment by piraartur** [comment]
> Sorry I think I've been unclear.
> java. 17 comes in addition to java 11 not as a replacement 

Oh, so you want me to add additional links and fields in table?

**Review by mteodori — APPROVED** [review]

**Review by alxgomz — APPROVED** [review]

**Pull request merged**